static/app/views/settings/account/apiApplications/index.spec.tsx

```diff
@@ -12,26 +18,28 @@ describe('ApiApplications', function () {
     MockApiClient.clearMockResponses();
   });
 
-  it('renders empty', function () {
+  it('renders empty', async function () {
     MockApiClient.addMockResponse({
       url: '/api-applications/',
       body: [],
     });
 
     render(<ApiApplications {...routerProps} />);
+    await waitForElementToBeRemoved(() => screen.queryByTestId('loading-indicator'));
 
     expect(
       screen.getByText("You haven't created any applications yet.")
     ).toBeInTheDocument();
   });
 
-  it('renders', function () {
+  it('renders', async function () {
     const requestMock = MockApiClient.addMockResponse({
       url: '/api-applications/',
       body: [ApiApplicationFixture()],
     });
 
     render(<ApiApplications {...routerProps} />);
+    await waitForElementToBeRemoved(() => screen.queryByTestId('loading-indicator'));
```

Comment:
next time i'd pull the `expect(await screen.findByText('Adjusted Shrimp')).toBeInTheDocument();` above the `expect(requestMock).toHaveBeenCalled();` instead of checking the loading indicators